java/bundles/org.eclipse.set.basis/src/org/eclipse/set/basis/constants/ToolboxConstants.java

```diff
@@ -271,6 +271,11 @@ public interface CacheId {
 	 */
 	public static final String ETCS_TABLE_PART_ID_PREFIX = "org.eclipse.set.feature.table.etcs";
 
+	/**
+	 * The prefix of ETCS Supplement table part
+	 */
+	public static final String ESTW_SUPPLEMENT_PART_ID_PREFIX = "org.eclipse.set.feature.table.supplement-estw";
```

Comment:
```suggestion
	public static final String ESTW_SUPPLEMENT_PART_ID_PREFIX = "org.eclipse.set.feature.table.estw-supplement";
```

Better readability in my opinion and from a sorting perspective estw and estw-supplement are next to each other then

Reply:
@mariusheine At the moment use we String.startWith({TABLE_PREFIX) to find, which Category the Table belong to. Because of this when use `estw-supplement`, then the Table of ESTW Supplement will belong to ESTW 😄

Reply:
@TruongQuangSB Ok then it makes sense to have it like this :smile: